### PR TITLE
Update the Self Defence checks screen

### PR DESCRIPTION
**Pull request by ziccardi**
# Motivation

Issue [AEROGEAR-2966](https://issues.jboss.org/browse/AEROGEAR-2966)

- [x] The **percentage trust score** should not show decimal places (round up/down).
- [x] A check which makes the device more secure should display with the following icon: https://material.io/tools/icons/?icon=verified_user
- [x] A check which makes the device less secure should display with the following icon https://material.io/tools/icons/?icon=warning
- [x] Add a refresh button to the iOS, Xamarin and Cordova Apps. Very useful for demoing. It exists already in the Android App -copy layout and style from there.
- [x] If the score is below a value determined by a constant in the App (See https://github.com/aerogear/android-showcase-template/blob/master/app/src/main/java/com/aerogear/androidshowcase/features/device/DeviceFragment.java#L46), offer the user to exit the App - in place for Android, to be completed for iOS, Xamarin, Cordova.
- [x] The wording of the checks should change depending on whether they have "passed" or "failed". This is already done for the Android app and should be used for the others.
- [x] Update wording to new specification: "Your current device trust score <score-value> is below the specified target of <target value>, do you want to continue or exit the app?"

## Content

- To implement the tasks above, a decorator has been added for the SecurityChecks so that an `IsSecure` method gets added to the SecurityCheckResult class together with the `secure` and `unsecure` messages. This way the page that visualizes the results is not cluttered with an endless list of `if/else`
- To avoid changing the code for each time we want to change the messages, string localization has been setup and most of the strings has been localized.
- SecurityPage has been changed to be able to adaptively change itself when running in Android or iOS
- A new component has been used to implement the Floating Action Button in a platform independent manner

Below the screenshot of the implementations

## Android
screen shot 2018-06-12 at 16 32 22
screen shot 2018-06-12 at 16 32 48

## iOS
Security Check List with Popup
Security Check List

### CODE REVIEW

File: Example.Android/Example.Android.csproj [hunk 1]
```diff
@@ -16,6 +16,7 @@
     <MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
     <MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
     <AndroidUseLatestPlatformSdk>false</AndroidUseLatestPlatformSdk>
+    <AndroidUseSharedRuntime>false</AndroidUseSharedRuntime>
```
Comment (wojta) [inline Example.Android/Example.Android.csproj]: Do you know what are these changes all about?

**Comment by ziccardi** [comment]
@wojta Mind giving another look?

**Comment by wei-lee** [comment]
@ziccardi can we make sure the color of the refresh icon on ios is the same as on other platforms (which should be orange)

**Comment by ziccardi** [comment]
@wei-lee : Yep. I did extra work to make it grey 👍

**Pull request merged**